js/components/portal-dashboard/feedback/feedback-question-rows.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Map } from "immutable";
+import Answer from "../../../containers/portal-dashboard/answer";
+import striptags from "striptags";
+import { renderHTML } from "../../../util/render-html";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
```

Comment:
Is this for future use?  If so, we can comment out for now to eliminate the console warnings.